src/DependencyGraph/FormulaCellVertex.ts

```diff
@@ -241,10 +241,9 @@ export class FormulaCellVertex extends FormulaVertex {
     version: number,
   ) {
     super(formula, address, version)
-    this.cachedCellValue = null
   }
 
-  public valueOrNull(): InterpreterValue | null {
+  public valueOrNull(): Maybe<InterpreterValue> {
```

Comment:
Method name is `valueOrNull` but it's `undefined` now. Does it make sense or it doesn't matter?